Bottom: 43cd2eda282b23783d461aad5ab8b99c6752bfdd
Top:    109386ac25479151eb665e0706b899c45ffd0f76
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:10:25 +0000

Refresh of revert-ripple-out-step-emission-3

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 6a00a7d..aad155b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -302,7 +302,6 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
   *y = COORD_TO_MM (vertex->point[1]); /* FIXME: PCB's coordinate system has y increasing downwards */
 }
 
-
 void
 object3d_export_to_step (object3d *object, const char *filename)
 {
